teamB/cheonsongin/app.js

```diff
@@ -6,28 +6,14 @@ const morgan = require('morgan');
 const dotenv = require('dotenv');
 dotenv.config();
```

Comment:
- dotenv 호출시 아래와 같이 수정해주시고, 최상단에 작성해주세요 ~!
```suggestion
require('dotenv').dotenv.config();
```